### PR TITLE
Update engine_cart to 1.x

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,14 +2,41 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'rspec-its'
+# Temporary hack to troubleshoot what elrayle is seeing
+gem 'rails', '~> 5.0.2'
```
Comment (mjgiarlo) [inline Gemfile]: Advice here, @jcoyne?

Comment (jcoyne) [inline Gemfile]: I do not know what @elrayle is seeing.

**Comment by elrayle** [comment]
I tested this locally and it works for me as well.

File: Gemfile (hunk 1 above)
Comment (elrayle) [inline Gemfile]: I tested this branch with rails 5.0.2 and 5.1.0.  Everything works fine at 5.0.2.  There are incompatible dependencies at 5.1.0 when trying to generate engine_cart.

Comment (elrayle) [inline Gemfile]: Log from engine_cart:generate from failure to end of log (rails 5.1.0):  https://gist.github.com/elrayle/2cb05cba2963dd84635f2ccf956f79c9

**Pull request merged**